packages/core/src/codewhisperer/views/lineAnnotationController.ts

```diff
@@ -435,11 +426,13 @@ export class LineAnnotationController implements vscode.Disposable {
             return undefined
         }
 
-        const nextState = this._currentState.nextState({
-            isCWRunning: isCWRunning,
-            source: source,
-            recommendationCount: e?.response?.recommendationCount ?? 0,
-        })
+        let nextState: AnnotationState | undefined
+
+        if (force) {
+            nextState = this._currentState.nextState(source, true)
+        } else {
+            nextState = this._currentState.nextState(source, false)
+        }
```

Comment:
const nextState = this._currentState.nextState(source, force)

Reply:
Addressed here: https://github.com/aws/aws-toolkit-vscode/pull/4554